homeassistant/components/knx/climate.py

```diff
@@ -322,6 +354,43 @@ async def async_set_preset_mode(self, preset_mode: str) -> None:
             )
             self.async_write_ha_state()
 
+    @property
+    def fan_mode(self) -> str:
+        """Return the fan setting."""
+
+        fan_speed = self._device.current_fan_speed
+
+        if fan_speed is None:
+            return FAN_OFF
+
+        if fan_speed == 0:
+            return FAN_OFF
+
+        if self._attr_fan_modes is None:
+            return FAN_OFF
+
+        if self._device.fan_mode == FanSpeedMode.STEP:
+            return self._attr_fan_modes[fan_speed]
+
+        return f"{fan_speed}%"
```

Comment:
Have not seen this in UI yet, but should there be a space between the value and the %-sign?

Reply:
<img width="182" alt="Screenshot 2024-09-22 at 8 54 04" src="https://github.com/user-attachments/assets/f9d9fe7f-e071-4451-9e9c-1eb908fa48bb">

Looks good to me without the space.